unittests/API/NodeAPITest.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <gtest/gtest.h>
+#include <hermes/hermes.h>
+#include <cstdint>
+
+#include "js_native_api.h"
+#include "js_native_api_types.h"
```

Comment:
It must not be necessary to include "js_native_api_types.h". It is already referenced in the "js_native_api.h".